style.css

```diff
@@ -0,0 +1,239 @@
+/* GENERAL */
+
+*, *::before, *::after {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Josefin Sans', sans-serif;
+  font-weight: 300;
+  background-color: #FBF8F5;
+  color: #414042;
+}
+
+
+/* TEXT */
+
+h1 {
+  color: #414042;
+  text-align: center;
+  font-size: 50px;
+}
+
+h2 {
+  color: #414042;
+  text-align: center;
+  font-size: 25px;
+  margin-bottom: 30px;
+}
+
+h3 {
+  font-size: 20px;
+  line-height: 1.4em;
+}
+
+h3 span {
+  font-weight: 300;
+}
+
+h4 {
+  font-size: 18px;
+  margin: 20px 0px -5px 0px;
+}
+
+h5 {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 1.3em;
+  margin-bottom: 10px; 
+}
+
+p {
+  line-height: 1.2em;
+  margin: 0px;
+}
+
+
+/* HEADER */
+
+header {
+  height: 120px;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  padding: 0 20px;
+}
+
+.hamburger {
+  display: flex;
+  width: 30px;
+  height: 30px;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-right: 24px;
+}
+
+.hamburger div {
+  width: 30px;
+  height: 5px;
+  background: #414042;
+  border-radius: 5px;
+}
+
+header img {
+  width: 180px;
+  align-self: center;
+  margin-right: auto;
+}
+
+header a img {
+  height: 18px;
+  width: 18px;
+  display: none;
+  margin-left: 20px
+}
+
+header img:hover {
+  animation-name: scale;
+  animation-duration: 0.6s;
+}
+
+nav {
+  display: none; 
+}
+
+nav a {
+  color: #414042;
+  font-weight: 400;
+  text-decoration: none;
+  margin-left: 5px;
+  padding: 8px 10px;
+  border: 2px solid transparent;
+}
+
+nav a:hover {
+  border: 2px solid #8e8806;
+  border-radius: 5px;
+}
+
+
+
+/* MAIN CONTENT */
+
+.parent {
+  height: 100px;
+  display: flex;
+  align-items: stretch;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+.baby {
+  margin: 5px;
+  padding: 20px 20px;
+  width: 100%;
+  height: auto; 
+  object-fit: cover;
+  display: flex;
+  border-radius: 5px;
+  color: #F5F2F0;
+  align-items: flex-start;
+  justify-self: center;
+  flex-direction: column;
+}
+
+.darkred {
+  background: #6c0826;
+}
+.olive {
+  background: #8e8806;
+}
+.red {
+  background: #9f0f26;
+}
+.brown {
+  background: #935b07;
+}
+.none {
+  padding: 0; 
+}
+
+.none img {
+  width: 100%;
+  border-radius: 5px;
+}
```

Comment:
To make the images fill your whole container, add `height: 100%;` as well as `object-fit: cover;` and you should see a difference!